packages/xy-chart/src/series/ViolinPlotSeries.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Group } from '@vx/group';
+import { ViolinPlot } from '@vx/stats';
+import themeColors from '@data-ui/theme/build/color';
+
+import { callOrValue } from '../utils/chartUtils';
+
+import { violinPlotSeriesDataShape } from '../utils/propShapes';
+
+
+const propTypes = {
+  data: violinPlotSeriesDataShape.isRequired,
+  label: PropTypes.string.isRequired,
+
+  // attributes on data points will override these
+  fill: PropTypes.oneOfType([PropTypes.func, PropTypes.string]),
+  stroke: PropTypes.oneOfType([PropTypes.func, PropTypes.string]),
+  strokeWidth: PropTypes.oneOfType([PropTypes.func, PropTypes.number]),
+
+  // likely be injected by the parent chart
+  xScale: PropTypes.func,
+  yScale: PropTypes.func,
+  horizontal: PropTypes.bool,
+  widthRatio: PropTypes.number,
+};
+
+const defaultProps = {
+  boxWidth: null,
+  stroke: '#000000',
```

Comment:
I would use `colors.darkGray` or `colors.black` which aren't quiiiite black. most designers have told me to never use pure black ... some dark gray instead is more visually appealing.